### DISCUSSION TITLE
Procedure to Request a New Distribution

### DISCUSSION THREAD

**Discussion by williamblair333** [antonym/netboot.xyz#1319] [General]
Hello, who do I speak with regarding the addition of a new distribution?  I was looking for MX Linux and didn't see it.  Would the maintainers need to contact you?

**Comment by antonym** [comment]
I think we looked at MX Linux a while back (https://github.com/netbootxyz/netboot.xyz/issues/348), but it wasn't very netboot friendly at the time. I'm more than willing to add it if there's a usable way of booting it over the network.

**Reply by williamblair333** [reply]
Would you point me to some documentation on how you guys get distributions into your 'ecosystem'?  I think these are it?  https://netboot.xyz/docs/faq/ [https://github.com/netbootxyz/build-pip ... /README.md](https://github.com/netbootxyz/build-pipelines/blob/master/README.md)  I appreciate the time.
Also - and this going off topic - but - could the MX distro it be mounted over an nfs volume?  I ask because I attempted already and failed; although it's entirely possible I'm very ignorant and did something wrong.  

Thanks,
